ultralytics/engine/exporter.py

```diff
@@ -356,8 +354,8 @@ def export_onnx(self, prefix=colorstr("ONNX:")):
         requirements = ["onnx>=1.12.0"]
         if self.args.simplify:
             requirements += ["onnxsim>=0.4.33", "onnxruntime-gpu" if torch.cuda.is_available() else "onnxruntime"]
-            if IS_RASPBERRYPI or IS_JETSON:
-                check_requirements("cmake")  # 'cmake' is needed to build onnxsim on Raspberry Pi and Jetson
+            if ARM64:
```

Comment:
For example, here we do not need `cmake` for macos ARM. So we are doing an unnecessary `cmake` install for macos ARM.

Reply:
What about users running aarch64 linux docker images on macos devices?

Reply:
I think we may want to selectively replace ARM64 with more specific tuples/lists of devices, but we have a wholesale search and replace going on here in places like exporter.py that I don't think is warranted right now, especially as this leaves us more brittle to future aarch64 devices that users may encounter in the future that the PR will no longer recognize as aarch64 environments.

Reply:
You got a point there. Let's keep it as it is then. I did not think about the whole picture here.